### PR TITLE
chore: simplify dev workflow

### PR DESCRIPTION
**Pull request by eysi09**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -58,29 +58,68 @@ Install Node modules for the root package, and the `dashboard` and `garden-servi
...
+```sh
+cd garden-service
+npm test -- -g "taskGraph"
+```
+
+Integration tests are run with:
+
+
+```sh
+npm integ
```
Comment (edvald) [inline CONTRIBUTING.md]: This should be `npm run integ`, right?

File: package.json [hunk 2]
```diff
@@ -48,30 +48,22 @@
   },
   "scripts": {
     "build": "npm run clean && gulp build",
-    "bootstrap": "npm install && cd garden-service && npm install && cd .. && cd dashboard && npm install && cd ..",
+    "bootstrap": "lerna boostrap",
```
Comment (edvald) [inline package.json]: Typo

Comment (eysi09) [reply package.json]: 9/10 times I mess this one up.

**Comment by eysi09** [comment]
I've resolved the comments.

I also removed the `gulpfile.ts` from the dashboard directory. It's currently not possible to run it because Create React App generates a tsconfig with options that break the import statements. Namely, `compilerOptions.mode` is set to `"esnext"`. It's not possible to manually remove it because the `compilerOptions` are re-set on every build.

**Pull request merged**